experiments/exp923_medu_mmlu_science_rewrite.py

```diff
@@ -0,0 +1,68 @@
+"""Synthetic data generation to rewrite high quality science data into QA format."""
```

Comment:
Link to GitHub issue, and a little bit more context / summary of what we did here.